server/validation/is-empty.js

```diff
@@ -0,0 +1,7 @@
+const isEmpty = value => (
+  value === undefined
+      || value === null
+      || (typeof value === 'object' && Object.keys(value).length === 0)
+      || (typeof value === 'string' && value.trim().length === 0)
+);
```

Comment:
This is good. I think the case for an empty array should be included here. we might not need it but i think it should be in here as well just in case someone needs is.